integration_tests/integration/link/emailSent.spec.ts

```diff
@@ -1,33 +1,35 @@
+import { skipOn } from '@cypress/skip-test'
 import Page from '../../pages/page'
 import RequestLinkPage from '../../pages/link/requestLink'
 import EmailSentPage from '../../pages/link/emailSent'
+import featureFlags from '../../support/featureFlags'
 
-context('Link Email Sent Page', () => {
-  beforeEach(() => {
-    cy.task('reset')
-    cy.task('stubAuthToken')
-    cy.task('stubRequestLink')
-  })
+skipOn(featureFlags.isLsjOneTimeCodeAuthEnabled(), () =>
```

Comment:
Whilst the whole file looks like its changed, really the only change is wrapping the `context` call in `skipOn(featureFlags.isLsjOneTimeCodeAuthEnabled(),` (which affects the indentation of the rest of the file, hence the change)
All the tests in this folder (`integration/link`) are related to magic link auth (there are 3 files with this exact same change). Because they are magic link based they dont work if the feature flag for One Time Codes is set, hence this `skipOn` logic